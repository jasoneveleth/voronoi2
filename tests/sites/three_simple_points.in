0.875, 0.169
0.852, 0.792
0.233, 0.434
