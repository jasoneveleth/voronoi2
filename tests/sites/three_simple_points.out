(0.875000,0.169000)	(0.852000,0.792000)	(0.233000,0.434000)	
(0.318678,1.000000),(0.624240,0.471667)	(0.624240,0.471667),(1.000000,0.485539)	(0.429549,-0.000000),(0.624240,0.471667)	
